README.md

```diff
@@ -121,13 +121,14 @@ module Repository
 
     module Wrapper
       class Validator
-        def before_find(id:)
+        def find(id:)
           raise ArgumentError if id.nil?
-          {id: id}
+          yield(id: id)
         end
       end
       class ModelMapper
-        def after_find(return_value, **_kwargs)
+        def find(id:)
+          return_value = yield(id: id)
           Model::User.new(return_value)
```

Comment:
would be nice to have the full example consistent with the wrapper part above ;-)
 
```suggestion
          Model::User.new(yield(id: id))
```